src/components/completer-cmp.ts

```diff
@@ -134,12 +134,14 @@ export class CompleterCmp implements OnInit, ControlValueAccessor, AfterViewChec
     @Input() public inputClass: string;
     @Input() public autofocus = false;
     @Input() public openOnFocus = false;
+    @Input() public openOnClick = false;
```

Comment:
please also add a description to the README and a sample of using it to the demo